hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java

```diff
@@ -93,16 +93,16 @@ public final class ContainerBalancerConfiguration {
       "to exclude from balancing. For example \"1, 4, 5\" or \"1,4,5\".")
   private String excludeContainers = "";
 
-  @Config(key = "move.timeout", type = ConfigType.TIME, defaultValue = "30m",
+  @Config(key = "move.timeout", type = ConfigType.TIME, defaultValue = "60m",
```

Comment:
Since the replication commands scheduled by `MoveManager` as part of `MoveManager#move` have lower priority compared to replication commands by `ReplicationManager`, I've increased this default to 60 minutes.